### DISCUSSION TITLE
Is there any way to 'hide' system components again after marking them as removeable before?

### DISCUSSION THREAD

**Discussion by hanzbadua** [Fs00/Win10BloatRemover#3] [Questions] [answered]
Title

**Comment by Fs00** [comment, marked as answer]
If I understood correctly, your question is about hiding system apps that have been made removable in the list of apps in Settings app. In that case, the answer is no.
The only way would be to make them un-removable again. In order to do this, the tool would need to keep track of which system apps have been made removable, which is impractical and imho not worth the effort.

**Reply by hanzbadua** [reply]
Ah, alright - thanks for the reply

**Comment by r3d1r** [comment]
Hi.
Ok. I understood what Fs00 said...
But if I want to revert it one by one, by myself, could you please point me the basics of what to do ?

**Reply by Fs00** [reply]
I'd advise you against doing so... you would need to edit a SQLite database in a protected system directory using a tool such as [DB Browser for SQLite](https://sqlitebrowser.org/), which is a quite complicated and error-prone process when done manually.
The simplest and most effective thing you can do is to install a Windows cumulative update, which will make most (but not all) system apps non-removable again.
The next version of the tool (which I hope to release before the summer) will feature a new mechanism for removing system apps that won't require users to make all of them removable.
